### PR TITLE
Update xboxdrvstart.sh SCUMMVM

### PR DESCRIPTION
**Pull request by bebeidon**
i added back+start=KEY_LEFTCTRL+KEY_Q,start+back=KEY_LEFTCTRL+KEY_Q,
becasue it seemed exiting only worked when pressing select first and then additonally start. if you'd happen to press start first and then select it wouldn't exit.

Also i read in the scummvm documentation, that if the game does not present the scummvm menu when pressing R like in the monkey island series (all games that were initally programmed in SCUMM it seems) you should only use the save and load options given by the game. forcing to do otherwise can actually break the game.
in conclusion i removed the save/load hotkey function.

### CODE REVIEW

**Comment by SinisterSpatula** [comment]
unable to merge due to conflict.
